crates/pyhq/python/hyperqueue/client.py

```diff
@@ -1,14 +1,15 @@
 from pathlib import Path
 from typing import Optional
 
-from .ffi.ffi import connect_to_server, submit_job
+from .ffi.connection import Connection
 from .job import Job, JobId
 
 
 class Client:
-    def __init__(self, path: Optional[Path] = None):
+    def __init__(self, path: Optional[Path] = None, python_bin="python3"):
```

Comment:
It will be useful to pass virtualenv here. Is it enough to pass e.g. `venv/bin/python`? Or does the environment need to be sourced?

Reply:
Good idea to pass venv. Just python is probably not enough. I will investigate it tomorrow